src/components/Visualization/Visualization.js

```diff
@@ -104,7 +105,10 @@ export class UnconnectedVisualization extends Component {
         this.props.setLoadError(error)
     }
 
-    onChartGenerated = (svg) => this.props.setChart(svg)
+    onChartGenerated = (svg, highchartsChart) => {
+        this.props.setChart(svg)
```

Comment:
I guess this is still needed for SV, correct?

Reply:
I wonder if we can just pass `highchartsChart` and call `getSVGForExport` for `setChart` here.